public/app/features/alerting/unified/components/contact-points/ContactPoints.tsx

```diff
@@ -99,7 +126,8 @@ const ContactPointsTab = () => {
           )}
         </Stack>
       </Stack>
-      <ContactPointsList contactPoints={contactPoints} search={search} pageSize={DEFAULT_PAGE_SIZE} />
+      {error && <Alert title="Failed to fetch contact points">{stringifyErrorLike(error)}</Alert>}
```

Comment:
Had to move the error so we still render action/filters list for the case where permissions don't let you see the list

Reply:
I believe we should still keep the error outside as it was, as in case of error, whats the point on seeing the action/filters ?

Reply:
I moved it here as I was trying to avoid the possibility of any other error stopping the rendering of the actions - at the time of writing, it was possible for the list endpoint to return a 403, just because you couldn't _see_ any contact points (but you had permission to create them). Are you okay if I leave it this way, to mitigate the risk of that kind of thing (unexpectedly) happening?